test/expected/domain_field_types.out

```diff
@@ -0,0 +1,1092 @@
+                     "fieldCharacterVarying": "ABCDE"        +
+                 }                                           +
+             ],                                              +
+             "affectedCount": 1                              +
+         }                                                   +
+     }                                                       +
+ }
+(1 row)
+
+rollback to savepoint a;
+-- Delete mutation, more a canary in a coal mine than anything else as the types are the same as the select
+select jsonb_pretty(
+  graphql.resolve(
+    $$
+    mutation deleteDomainTest {
+      deleteFromDomainTestCollection(
+        filter: {
+          id: {eq: 1}
+        }
+      ) {
+        affectedCount
+        records {
+          id
+          fieldInt
+          fieldSmallint
+          fieldBigint
+          fieldNumeric
+          fieldReal
+          fieldDoublePrecision
+          fieldBoolean
+          fieldText
+          fieldCharacter
+          fieldCharacterVarying
+          fieldDate
+          fieldTime
+          fieldTimestamp
+          fieldJsonb
+          fieldJson
+        }
+      }
+    }
+    $$
+  )
+);
+                         jsonb_pretty                         
+--------------------------------------------------------------
+ {                                                           +
+     "data": {                                               +
+         "deleteFromDomainTestCollection": {                 +
+             "records": [                                    +
+                 {                                           +
+                     "id": 1,                                +
+                     "fieldInt": 42,                         +
+                     "fieldDate": "2022-01-01",              +
+                     "fieldJson": "{\"b\": 2}",              +
+                     "fieldReal": 3.14,                      +
+                     "fieldText": "Hello, world!",           +
+                     "fieldTime": "12:34:56",                +
+                     "fieldJsonb": "{\"a\": 1}",             +
+                     "fieldBigint": "2165559898978",         +
+                     "fieldBoolean": true,                   +
+                     "fieldNumeric": "123.45",               +
+                     "fieldSmallint": 32767,                 +
+                     "fieldCharacter": "A",                  +
+                     "fieldTimestamp": "2022-01-01T12:34:56",+
+                     "fieldDoublePrecision": 3.14159265359,  +
+                     "fieldCharacterVarying": "ABCD"         +
+                 }                                           +
+             ],                                              +
+             "affectedCount": 1                              +
+         }                                                   +
+     }                                                       +
+ }
+(1 row)
+
```

Comment:
Could you add a tests that show what kind of error output we get if a user provides an invalid input value for a domain field?

For example, given a `pos_int` domain:
- what happens if the user enters a negative number in a Filter
- what happens if the user tries to insert a negative value

Reply:
Will do, Is this just for documenting behavior or is there a specific expected behavior other then returning:

```json
 { 
     "data": null,
     "errors": [ 
         {
             "message": "Something bad happened."
         }
     ]
 }
```

Reply:
just for documenting behavior, but if its too generic I might make a note to improve in the future